src/support/kuzzle.ts

```diff
@@ -68,8 +79,28 @@ export class KuzzleSDK {
         password: this.password,
       }
 
-      await this.sdk.auth.login('local', credentials, this.loginTTL)
-      log(chalk.green(`[ℹ] Loggued as ${this.username} for ${this.loginTTL}.`))
+      await this.sdk.auth.login('local', credentials, '90s')
+
+      if (this.refreshLogin) {
+        this.refreshTimer = setInterval(async () => {
+          try {
+            await this.sdk.auth.refreshToken()
+          }
+          catch (error) {
+            log(`Cannot refresh token: ${error}`)
+          }
+        }, ONE_SECOND)
+      }
+
+      log(chalk.green(`[ℹ] Loggued as ${this.username} (refreshLogin: ${this.refreshLogin}).`))
+    }
+  }
+
+  disconnect() {
+    this.sdk.disconnect()
+
+    if (this.refreshTimer) {
```

Comment:
(nitpicking) the behavior of `clearInterval` on null or undefined vaues is a no-op, so this if-condition is unnecessary

Reply:
I can't remove the condition otherwise Typescript will complain :expressionless: